setuptools/config/_validate_pyproject/fastjsonschema_validations.py

```diff
@@ -1,5 +1,4 @@
 # noqa
-# type: ignore
```

Comment:
This is vendored code, but the fix is already upstream: https://github.com/abravalheri/validate-pyproject/pull/159